src/components/pages/TalkPage/index.jsx

```diff
@@ -0,0 +1,386 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBCardFooter,
+  MDBBtn,
+  MDBBadge,
+  MDBIcon,
+  MDBPageItem,
+  MDBCardHeader,
+  MDBMedia,
+  MDBPageNav,
+  MDBPagination,
+} from "mdbreact";
+
+//> CSS
+import "./talk.scss";
+//#endregion
+
+//#region > Components
+class TalkPage extends React.Component {
```

Comment:
Add description: `This component adds the Talk Page which dipslays a certain talk.`